src/main/java/com/sda/strings/TextTruncate.java

```diff
@@ -0,0 +1,25 @@
+package com.sda.strings;
+
+public class TextTruncate implements StringOperation {
+    int length;
+
+    public TextTruncate(int length) {
+        this.length = length;
+    }
+
+    @Override
+    public String modify(String text) {
+        String resoult = "";
+        char[] tab = text.toCharArray();
+        if (tab.length < length) {
+            for (int i = 0; i < tab.length; i++) {
+                resoult += tab[i];
+            }
+        } else{
+            for (int i = 0; i < length; i++) {
+                resoult += tab[i];
+            }
+        }
+        return resoult;
```

Comment:
Jak wyżej, super że to zaimplementowałeś  i zadbałeś o warunki brzegowe. Z obowiązku powiem że można było też użyć metodę klasy `String` -  `substring`. Jeśli będziesz chciał zmienić swoją implementację na `substring` to pamiętaj o wzięciu pod uwagę dalej warunków brzegowych.